image-rs/src/resource/kbs/grpc.rs

```diff
@@ -24,31 +24,30 @@ mod get_resource {
 
 /// Attestation Agent's GetResource gRPC address.
 /// It's given <https://github.com/confidential-containers/guest-components/tree/main/attestation-agent#run>
-pub const AA_GETRESOURCE_ADDR: &str = "http://127.0.0.1:50001";
+///
+/// We are now using CDH to provide the GetResource gRPC service,
+/// while gRPC is still not supported by CDH. So this feature will
+/// not work.
+/// TODO: Add gRPC support for CDH to leverage the ability of getting
+/// resources by image-rs using gRPC.
+pub const GETRESOURCE_ADDR: &str = "http://127.0.0.1:50001";
```

Comment:
Yes. If gRPC is used it will break. Do you think we should temporarily comment out gRPC feature in Cargo.toml until gRPC is implemented?

BTW, luckily, currently the whole CoCo stack uses ttrpc and the feature [kata-cc-native-tls](https://github.com/confidential-containers/guest-components/blob/main/image-rs/Cargo.toml#L89) used by kata-agent sets ttrpc as default.

Reply:
Yeah might be good to disable the feature more noticeably. Could comment it out or maybe throw some compile-time error.

Reply:
Great. Let me add a compliation time error.